cmd/scenario.go

```diff
@@ -40,6 +44,59 @@ func newScenarioListCommand(logger *logrus.Logger) *cobra.Command {
 	return cmd
 }
 
+func newScenarioDescribeCommand(logger *logrus.Logger) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   `describe <id>`,
+		Short: "Describes a scenario",
+		Args:  cobra.MinimumNArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+
+			scenarioID := args[0]
+			manifestPath := viper.GetString("scenarios-dir")
+			manifest, err := scenario.LoadManifest(manifestPath)
+
+			if err != nil {
+				logger.WithFields(logrus.Fields{
+					"Error": err,
+				}).Error("Error loading scenario manifest")
+				return err
+			}
+
+			logger.WithFields(logrus.Fields{
+				"ScenarioID": scenarioID,
+			}).Debug("Checking manifest contains scenario")
+			if !manifest.Contains(scenarioID) {
+				return errors.Errorf("Scenario not found: %s", scenarioID)
+			}
+
+			scenario := manifest.Find(scenarioID)
+			err = scenario.Validate(manifestPath)
+			if err != nil {
+				return err
+			}
+
+			scenarioPath, err := filepath.Abs(filepath.Join(manifestPath, scenario.Path))
+			if err != nil {
+				return errors.Wrapf(err,
+					"Error resolving %s from %s for scenario %s", scenario.Path, scenario.DisplayName, manifestPath)
+			}
+
+			challengeContent, err := ioutil.ReadFile(scenarioPath + "/challenge.txt")
+			if err != nil {
+				return err
+			}
+			challengeText := string(challengeContent)
+
+			fmt.Println("Name: " + scenario.DisplayName)
```

Comment:
A newline after here would make this fit in with the format of the challenge.txt's